uaclient/messages.py

```diff
@@ -835,3 +837,7 @@ class TxtColor:
 Open a browser to: {}/subscribe""".format(
     BASE_UA_URL
 )
+
+NOTICE_REFRESH_CONTRACT_WARNING = """\
+A change has been detected in your contract.
+Please run `sudo ua refresh contract`."""
```

Comment:
I just realized my suggestion here is a bad idea. We should recommend a complete `sudo ua refresh` so that messages will be updated too. I'll just push a quick commit to this before merging.

Reply:
No worries.